docs/support/faq.md

```diff
@@ -68,7 +68,7 @@ Perform these steps to verify the media item has a guid Overseerr can match.
 
 **A:** The logs are located at `<Overseeerr-install-directory>/logs/overseerr.log`
 
-## User Management
+## User management
```

Comment:
Why the inconsistency of capitalizing these titles? I feel like all capitals here looks better? Just like the main title? `Frequently Asked Questions`. This goes for the other similar changes as well

Reply:
I noticed that many of the other subtitles were not capitalized, so I changed it.  But I can just capitalize them all instead :)